nautobot/utilities/choices.py

```diff
@@ -153,3 +153,73 @@ class ButtonColorChoices(ChoiceSet):
         (YELLOW, "Yellow"),
         (BLACK, "Black"),
     )
+
+
+class ButtonActionColorChoices(ChoiceSet):
+    """
+    Map standard button actions to Bootstrap color classes.
+    """
+
+    ADD = "success"
+    CANCEL = "default"
+    CLONE = "success"
+    CONFIGURE = "default"
+    DEFAULT = "default"
+    DELETE = "danger"
+    EDIT = "warning"
+    EXPORT = "success"
+    IMPORT = "primary"
+    INFO = "info"
+    SUBMIT = "primary"
+
+    CHOICES = (
+        (ADD, "Add"),
+        (CANCEL, "Cancel"),
+        (CLONE, "Clone"),
+        (CONFIGURE, "Configure"),
+        (DEFAULT, "Default"),
+        (DELETE, "Delete"),
+        (EDIT, "Edit"),
+        (EXPORT, "Export"),
+        (IMPORT, "Import"),
+        (INFO, "Info"),
+        (SUBMIT, "Submit"),
+    )
+
+
+class ButtonActionIconChoices(ChoiceSet):
+    """
+    Map standard button actions to Material Design Icons classes.
+    """
+
+    ADD = "mdi-plus-thick"
+    ALERT = "mdi-alert"
+    ARROW_DOWN = "mdi-arrow-down-bold"
+    ARROW_UP = "mdi-arrow-up-bold"
+    CONNECT = "mdi-ethernet-cable"
+    DISCONNECT = "mdi-ethernet-cable-off"
+    EDIT = "mdi-pencil"
+    HELP = "mdi-help-circle"
+    IMPORT = "mdi-database-import-outline"
+    LOCK = "mdi-lock"
+    MAGNIFY = "mdi-magnify"
+    NOTE = "mdi-note-text"
+    SWAP = "mdi-swap-vertical"
+    TRASH = "mdi-trash-can-outline"
```

Comment:
I'm curious where you selected this list of actions from? I would have naively expected that these would be the same set of keys as present in ButtonActionColorChoices, i.e. something more like

```python
ADD = "mdi-plus-thick"
CANCEL = "..."
CLONE = "..."
CONFIGURE = "mdi-cog"
DEFAULT = "..."
DELETE = "mdi-trash-can-outline"
EDIT = "mdi-pencil"
EXPORT = "mdi-database-export-outline"
IMPORT = "mdi-database-import-outline"
INFO = "mdi-help-circle"
SUBMIT = "..."
```

Though certainly adding more actions to both choicesets is not necessarily a bad idea; I'd just expect them to agree with one another about the actions included.

Reply:
Oh i see, sorry i did a search across all of nautobot to find the most common `mdi` icons. Should i remove the extras?

Reply:
Or i can make it a generic choice of icons, not just for the buttons?

Reply:
Not sure I follow your second comment but I'd be fine as I said with having both choicesets include more options, I just would expect them to have the same set of options more or less.